src/ResourceManager/Resources/Commands.Resources/help/Get-AzureRmManagementPartner.md

```diff
@@ -0,0 +1,98 @@
+---
```

Comment:
@jeffrey-ACE please move the `help` folder and its contents from the `Resources` project to the `ManagementPartner` project

Reply:
@cormacpayne will do